### PR TITLE
#143358917 Configure TravisCI

### PR DESCRIPTION
**Pull request by andela-onnenanya**
#### What does this PR do?
Integrate TravisCI with readme badge
#### Description of Task to be completed?
Integrate TravisCI with readme badge
#### How should this be manually tested?
Push a commit to a repository
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Integrate TravisCI with readme badge
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - 0.12
```
Comment (emmyakin) [inline .travis.yml]: can you bump this node version up to whatever version we are using, and update the readme

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+before_script:
+  - npm install -g gulp
+script: gulp
```
Comment (emmyakin) [inline .travis.yml]: you can omit this, defaults to gulp test, or use gulp test

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+install: true
+deploy:
+  skip_cleanup: true
+
+# safelist
+branches:
```
Comment (emmyakin) [inline .travis.yml]: we would need to run travis on all branches. remove this

**Review by emmyakin — CHANGES_REQUESTED** [review]
Check that travisCi runs after making the changes, grab the badge, and add to the readme

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - 6.10.2
+before_script:
+  - npm install -g gulp
+install: true
```
Comment (emmyakin) [inline .travis.yml]: remove this too

**Review by emmyakin — CHANGES_REQUESTED** [review]
I am not sure the rebase off develop worked, you still have README.md changes

File: .travis.yml (hunk 4 above)
Comment (andela-onnenanya) [reply .travis.yml]: Pls, do you mean I should just have lines 1 and 2?

Comment (andela-onnenanya) [reply .travis.yml]: Also, for the rebase, can I reclone the project and do a fresh stuff?

Comment (emmyakin) [reply .travis.yml]: I meant remove install: true, that setting means travis should not install node_modules
I suggest reading this - https://docs.travis-ci.com/user/customizing-the-build

**Review by emmyakin — APPROVED** [review]
All done. Thanks

**Pull request merged**